app/views/partials/_user.html.erb

```diff
@@ -0,0 +1,7 @@
+<section class="user_header d_flex">
+  <%= image_tag @user.photo, class: "user_img", size: "160x90", alt: "Profile Picture"%>
+  <div class=" user_info d_flex">
+    <h1><%= link_to @user.name %></h1>
+    <p>Number of posts: <%= @user.post_counter %></p>
+  </div>
+</section>
```

Comment:
- [ ] On the user, page ı should be able to see the 'see all post' button and then click on it, page redirects to the this [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_all_posts.png).
Kİndly fix it.
![image](https://user-images.githubusercontent.com/38635158/185517378-cad36f1b-d23d-4a12-97c8-07c884913d5e.png)